assets/js/open-profile-modal.js

```diff
@@ -0,0 +1,8 @@
+import {MDCDialog} from '@material/dialog';
+const dialog = new MDCDialog(document.querySelector('.mdc-dialog'));
+
+const openProfileModal = document.querySelector('#show').addEventListener('click', () => {
```

Comment:
I think this will cause a JS error on pages where that element isn't available.

Also I suggest for something as important as an ID, use a clearer and more unique name, like #show-business-menu